TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -362,7 +373,11 @@ inline typename ProjectionRouterBuffer<VMProjType, AllProjectionType>::TRKID get
   return (tcid_, allproj.getTrackletIndex());
 }
 
-inline ap_uint<kNBits_MemAddr> getProjSeq() {
+//
+// Old implemenation of the getProjSeq. This method can be
+// used instead of getProjSeq which now makes use of cached data
+//
+inline ap_uint<kNBits_MemAddr> getProjSeqOld() {
```

Comment:
Is this just a temporary function, or does something still use the old method?

Reply:
It is not used in the code, but I left it in case we wanted to go back and run without the caching for some reason.